### PR TITLE
Blogpost about our experience at Strange Loop

### PR DESCRIPTION
**Pull request by emkosz**

### CODE REVIEW

File: blog/_posts/2015-09-30-Cocoagems-gets-strange-at-Strange-Loop.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<br><font color="grey"><small><i> The pre-party was held at the City Museum. Photo CocoaGems.</i></small></font>
+
+## Take Aways
+
+A main take away from Strange Loop was the great value of connecting computer science with other interests or subjects. [Carin Meier](http://www.thestrangeloop.com/2015/unconventional-programming-with-chemical-computing.html) played around with building programs in Clojure based on chemical reactions. Since all living organisms information systems are based on chemical processes there is a lot we can learn from it and by showing the audience here demos she wanted to give an eye opener for more innovation and new ways of thinking about computer problems. [Amy Wibows](http://www.thestrangeloop.com/2015/sweaters-as-a-service.html) talk about how she and a a group of Airbnb engineers hacked a knitting machine to be computer-controlled so that it could print images in yarn. She and here team learned a great deal of electronics using the problem solving skills that computer science requires in a engineering day job. By applying these skills to a hobby project they experienced the joy of being new to something as well as getting yarn print outs.
```
Comment (alicetragedy) [inline blog/_posts/2015-09-30-Cocoagems-gets-strange-at-Strange-Loop.md]: Amy Wibowo

File: blog/_posts/2015-09-30-Cocoagems-gets-strange-at-Strange-Loop.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+<img src="/img/blog/2015/CG_SL_chemical.JPG" alt="Team CocoaGems">
+<br><font color="grey"><small><i> Carin Meier talks about Chemical Computing. Photo CocoaGems.</i></small></font>
+
+<img src="/img/blog/2015/CG_SL_knitting.JPG" alt="Team CocoaGems">
```
Comment (alicetragedy) [inline blog/_posts/2015-09-30-Cocoagems-gets-strange-at-Strange-Loop.md:24]: Amy Wibowo

**Comment by alicetragedy** [comment]
This is a great and insightful post @emkosz – thank you so much. As we have quite a few team scheduled posts over the next few days, I will merge this once we have a little bit more breathing room (probably next week). I hope that's okay. I will also change the date accordingly!
I've noticed a couple of spaces missing, and a speaker name misspelled. I understand that it's the end of the program though ;) so I can correct that when I merge next week – just wanted to make a note to myself, so I don't forget.

**Comment by alicetragedy** [comment]
### Note: will be merged in Week 41

File: blog/_posts/2015-09-30-Cocoagems-gets-strange-at-Strange-Loop.md (hunk 2 above)
Comment (emkosz) [inline blog/_posts/2015-09-30-Cocoagems-gets-strange-at-Strange-Loop.md:24]: Fixed

**Comment by emkosz** [comment]
Hi @alicetragedy Thank you for pointing that out. Have fixed the misspelled name and added spaces. Looking forward to reading the other teams blogposts. No rush in posting ours :)

**Comment by alicetragedy** [comment]
great, thanks @emkosz! We'll try to find your blog post a spot next week ;) In the meanwhile – enjoy your freedom ;)

**Pull request merged**